app/src/main/res/values/dimens.xml

```diff
@@ -6,7 +6,45 @@
     <dimen name="space_medium">20dp</dimen>
     <dimen name="space_large">32dp</dimen>
     <dimen name="space_extra_large">64dp</dimen>
-
+    <dimen name="dp_0">0dp</dimen>
```

Comment:
This is unnecessary work. Take a reference of above names of dimens. Write some practical names which you can relate to.

Reply:
Removed that Size specific named format.
Will use practical descriptive names from now on as you suggested.